src/test/regress/expected/failure_create_distributed_table_non_empty.out

```diff
@@ -0,0 +1,520 @@
+-- 
+--  Failure tests for COPY to reference tables 
+-- 
+CREATE SCHEMA create_distributed_table_non_empty_failure;
+SET search_path TO 'create_distributed_table_non_empty_failure';
+SELECT citus.mitmproxy('conn.allow()');
+ mitmproxy 
+-----------
+ 
+(1 row)
+
+-- we'll start with replication factor 1
```

Comment:
It seems that you're using 2PC. You should put that somewhere here in the header (with a `SET` command) so that this test doesn't depend on the ordering of tests, and also as documentation. It'd be nice if the file name also included "2pc" somewhere in it, so we know that that's the only thing you're testing.